lib/expressroutes.js

```diff
@@ -151,6 +175,17 @@ function expressroutes(router, options) {
         res.json(options.api);
     });
 
+    var routesMethod = {};
+    routes.forEach(function(route) {
```

Comment:
I would put this into the makeExpressRoutes  to avoid going through the routes list twice.